app/views/collections/_edit_slides_in_collection.html.erb

```diff
@@ -49,10 +49,17 @@
                   <%= builder.select('slide_type_id', options_from_collection_for_select(slide_types, :id, :name, builder.object.slide_type_id), { :include_blank => '-- Select One --' }, :required => true, class: "form-control") %>
                 </div>
 
+                <%= builder.label :kiosks %>
                 <div class="field">
-                  <%= builder.label :kiosk_id %>
-                  <%= builder.select('kiosk_id', options_from_collection_for_select(kiosks, :id, :name, builder.object.kiosk_id), { :include_blank => '-- Select One --' }, :required => true, class: "form-control") %>
+                  <ul class="list-group">
+                    <span class="ul">
+                      <% Kiosk.all.each do |kiosk| %>
```

Comment:
You should set a variable on the controller for `Kiosk.all` instead of querying in the view.